requirements_dev.txt

```diff
@@ -4,3 +4,6 @@ recommonmark==0.7.1
 # with regard to how it emits class names and em dashes from
 # time to time:
 Sphinx==3.5.4
+markupsafe==2.0.1
+Jinja2>2.0,<3.0
```

Comment:
on #191 the first change I make is updating to Jinja 3+, which necessitates an update to Sphinx 4+.

I agree that these pins should be a temporary measure for getting to a working build to make it easier to fix the problems that come from updating them to current versions of the library. As I put it on my review of the [initial PR](https://github.com/mozilla/sphinx-js/pull/195#discussion_r862082996):

> Jinja expressly advises upgrading Jinja instead of pinning, which I've been working on in https://github.com/mozilla/sphinx-js/pull/191 but its a big thorny mess. I'm ok with pinning here to get the build working again and I'll follow as quickly as I can with getting Jinja, Sphinx, and Parsimonious updated